cli/metrics/generatecommands/main.go

```diff
@@ -19,20 +19,26 @@ package main
 import (
 	"fmt"
 	"os/exec"
+	"sort"
 	"strings"
 
 	"github.com/docker/compose/v2/pkg/utils"
 )
 
-var managementCommands = []string{"ecs", "scan"}
-
-var commands = []string{}
+var (
+	managementCommands = []string{"ecs", "scan"}
+	commands           = []string{"bundle", "completion", "install", "merge", "render", "split", "status", "uninstall", "validate"}
```

Comment:
I'm not sure I understand why we need this... should we define buildx (& compose) as management commands so that we walk through subcommands ?

Reply:
somehow these commands were missing after the generation. see the specific commit: https://github.com/docker/compose-cli/pull/2104/commits/a4ae60a39d763cb4938396960bc3f8a8db26fe1b